tock/hours/models.py

```diff
@@ -236,8 +258,47 @@ def save(self, *args, **kwargs):
         """
         if not self.id and self.user:
             self.billable_expectation = self.user.user_data.billable_expectation
+        self.calculate_hours()
         super().save(*args, **kwargs)
 
+    def calculate_utilization_denominator(self):
+        return self.billable_hours + self.non_billable_hours
```

Comment:
Wondering if we could maybe leverage notifications framework to throw a message if this is > 40.

Reply:
Definitely possible, let's tackle that in a separate issue though if desired.